src/main/java/org/gitlab4j/api/HealthCheckApi.java

```diff
@@ -0,0 +1,80 @@
+package org.gitlab4j.api;
+
+import org.gitlab4j.api.models.LivenessHealthCheck;
+import org.gitlab4j.api.models.ReadinessHealthCheck;
+
+import javax.ws.rs.core.Response;
+import java.io.IOException;
+import java.net.URL;
+
+public class HealthCheckApi extends AbstractApi {
+    public HealthCheckApi(GitLabApi gitLabApi) {
+        super(gitLabApi);
+    }
+
+    /**
+     * Get Health Checks from the liveness endpoint.
+     * Requires ip_whitelist
+     *
+     * GET /-/liveness
+     *
+     * @return LivenessHealthCheck instance
+     * @throws GitLabApiException if any exception occurs
+     */
+    public LivenessHealthCheck getLiveness() throws GitLabApiException, IOException {
+        URL livenessUrl = getApiClient().getUrlWithBase("-", "liveness");
+        Response response = get(Response.Status.OK, null, livenessUrl);
+        return (response.readEntity(LivenessHealthCheck.class));
+    }
+
+    /**
+     * Get Health Checks from the liveness endpoint.
+     *
+     * GET /-/liveness
+     *
+     * @param token Health Status token
+     * @return LivenessHealthCheck instance
+     * @throws GitLabApiException if any exception occurs
+     * @deprecated
```

Comment:
Gitlab docs state these have been deprecated. I added them in any way so that it is at least backward compatible during the deprecation process.

Reply:
@lamdaV 
There is no good reason to add methods to deprecated endpoints.  A method should only be deprecated if it was already implemented and is being replaced.  Since the method is brand new in this library, no one has ever used it and no need to add it, do not wish to encourage people to use deprecated methods.